app/assets/stylesheets/layouts/_results.scss

```diff
@@ -25,3 +25,28 @@
     float: right;
   }
 }
+
+.l-results__list {
+  @extend .unstyled-list;
+}
+
+.l-results__pagination {
+  @include column(12);
+  margin-top: $baseline-unit*4;
+  margin-bottom: $baseline-unit*6;
+
+  @include respond-to ($mq-m) {
+    @include column(6);
+    margin-bottom: $baseline-unit*8;
+  }
+}
+
+.l-back-to-top {
```

Comment:
Is this generic enough to live in its own layout file? If not, should it be a child of `l-results`?